meta-balena-common/conf/distro/include/balena-os.inc

```diff
@@ -146,6 +146,7 @@ BALENA_BOOT_MOUNT ?= "/mnt/${@'${BALENA_BOOT_LABEL}'.replace('balena-','')}"
 BALENA_SIGN_LABEL ?= "${@bb.utils.contains('MACHINE_FEATURES', 'efi', 'balena-efi', "${BALENA_BOOT_LABEL}",d)}"
 BALENA_NONENC_BOOT_LABEL ?= "${@oe.utils.conditional('SIGN_API','','${BALENA_BOOT_LABEL}','${BALENA_SIGN_LABEL}',d)}"
 BALENA_NONENC_BOOT_MOUNT ?= "/mnt/${@'${BALENA_NONENC_BOOT_LABEL}'.replace('balena-','')}"
+BALENA_USE_LUKS ?= "1"
```

Comment:
Who reads this?

Reply:
It's the value for `USE_LUKS`:
```
echo "USE_LUKS=${BALENA_USE_LUKS}" >> ${D}/${sysconfdir}/resin-init-flasher.conf
```
It is then overridden by device repositories, in the case of the i.MX by the `meta-balena-hab` layer configuration file.